components/organisms/UserSettingsPage/user-settings-page.tsx

```diff
@@ -227,7 +228,14 @@ const UserSettingsPage = ({ user }: userSettingsPageProps) => {
                 placeholder="Tell us about yourself."
                 className="px-3 py-2 rounded-lg bg-light-slate-4 disabled:cursor-not-allowed "
                 name="bio"
+                value={bio}
+                onChange={(e) => setBio(e.target.value)}
               ></textarea>
+              {bio?.length > 255 ? (
+                <p className="text-light-red-10 text-xs">Bio too long</p>
+              ) : (
+                <p className="text-xs">{bio?.length}/255</p>
+              )}
```

Comment:
refactor (blocking): `aria-live="assertive"` should be used for the validation error and `"polite"` for the character count. See https://www.smashingmagazine.com/2023/02/guide-accessible-form-validation/#instant-validation and https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions

```suggestion
              {bio?.length > 255 ? (
                <p aria-live="assertive" className="text-light-red-10 text-xs">Bio too long</p>
              ) : (
                <p aria-live="polite" className="text-xs">{bio?.length}/255</p>
              )}
```